src/app/components/MeasurementMassPlot.tsx

```diff
@@ -94,19 +92,19 @@ function MassComponent(props: MeasurementPlotProps) {
         <LineSeries
           data={profile}
           lineStyle={{ stroke: 'green' }}
-          xAxis={xAxis}
-          yAxis={yAxis}
+          xAxis="x"
+          yAxis="y"
```

Comment:
xAxis is "x" by default, and yAxis is "y" by default. Please remove these props everywhere unless a different value is passed.

Reply:
fixed